reference/speeduino.ini

```diff
@@ -524,47 +524,32 @@ page = 9
-        unused10_3 = scalar, U08,      3,         "",       1, 0, 0, 255, 0
-        unused10_4 = scalar, U08,      4,         "",       1, 0, 0, 255, 0
-        unused10_5 = scalar, U08,      5,         "",       1, 0, 0, 255, 0
-        unused10_6 = scalar, U08,      6,         "",       1, 0, 0, 255, 0
-        unused10_7 = scalar, U08,      7,         "",       1, 0, 0, 255, 0
-        unused10_8 = scalar, U08,      8,         "",       1, 0, 0, 255, 0
-        unused10_9 = scalar, U08,      9,         "",       1, 0, 0, 255, 0
-        unused10_10 = scalar, U08,     10,        "",       1, 0, 0, 255, 0		
-        unused10_11 = scalar, U08,     11,        "",       1, 0, 0, 255, 0
-        unused10_12 = scalar, U08,     12,        "",       1, 0, 0, 255, 0
-        unused10_13 = scalar, U08,     13,        "",       1, 0, 0, 255, 0
-        unused10_14 = scalar, U08,     14,        "",       1, 0, 0, 255, 0
-        unused10_15 = scalar, U08,     15,        "",       1, 0, 0, 255, 0
-        unused10_16 = scalar, U08,     16,        "",       1, 0, 0, 255, 0
-        unused10_17 = scalar, U08,     17,        "",       1, 0, 0, 255, 0
-        unused10_18 = scalar, U08,     18,        "",       1, 0, 0, 255, 0
-        unused10_19 = scalar, U08,     19,        "",       1, 0, 0, 255, 0
-        unused10_20 = scalar, U08,     20,        "",       1, 0, 0, 255, 0
-        unused10_21 = scalar, U08,     21,        "",       1, 0, 0, 255, 0
-        unused10_22 = scalar, U08,     22,        "",       1, 0, 0, 255, 0
-        unused10_23 = scalar, U08,     23,        "",       1, 0, 0, 255, 0
-        unused10_24 = scalar, U08,     24,        "",       1, 0, 0, 255, 0
-        unused10_25 = scalar, U08,     25,        "",       1, 0, 0, 255, 0
-        unused10_26 = scalar, U08,     26,        "",       1, 0, 0, 255, 0
-        unused10_27 = scalar, U08,     27,        "",       1, 0, 0, 255, 0
-        unused10_28 = scalar, U08,     28,        "",       1, 0, 0, 255, 0
-        unused10_29 = scalar, U08,     29,        "",       1, 0, 0, 255, 0
-        unused10_30 = scalar, U08,     30,        "",       1, 0, 0, 255, 0
-        unused10_31 = scalar, U08,     31,        "",       1, 0, 0, 255, 0
-        unused10_32 = scalar, U08,     32,        "",       1, 0, 0, 255, 0
-        unused10_33 = scalar, U08,     33,        "",       1, 0, 0, 255, 0
-        unused10_34 = scalar, U08,     34,        "",       1, 0, 0, 255, 0
-        unused10_35 = scalar, U08,     35,        "",       1, 0, 0, 255, 0
-        unused10_36 = scalar, U08,     36,        "",       1, 0, 0, 255, 0
-        unused10_37 = scalar, U08,     37,        "",       1, 0, 0, 255, 0
-        unused10_38 = scalar, U08,     38,        "",       1, 0, 0, 255, 0
-        unused10_39 = scalar, U08,     39,        "",       1, 0, 0, 255, 0
-        unused10_40 = scalar, U08,     40,        "",       1, 0, 0, 255, 0
+        enable_candata_in = bits,   U08,     0, [0:0], "Off", "On"

```

Comment:
If these are simple on/off bits, then they are being represented incorrectly in these ini lines. The memory address (0 in this case) should not increment for each bit as that are all part of the same overall byte. 

Have a look at the 'bits' entries on the other pages and see if that's how you were trying to use them.